modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,62 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
 == Required account privileges
 
-Installing a cluster to Nutanix requires an account with administrative privileges to read and create the required resources.
+The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster, as well as to maintain the daily operation of it. The following options are available to you:
```

Comment:
```suggestion
The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster and to maintain daily cluster operations. The following options are available to you:
```
![Screenshot from 2023-08-02 14-47-55](https://github.com/openshift/openshift-docs/assets/57954076/0c6dc724-0b70-4215-9acd-1e81f1bacd70)

It is a vague referent that could refer to closer/permissions/account.

Reply:
Yep. Thank you.

Reply:
:heavy_check_mark: